defaults/parameters.yaml

```diff
@@ -115,7 +115,8 @@ combine_sequences_for_subsampling:
   warn_about_duplicates: true
 
 tree:
-  tree-builder-args: "'-ninit 10 -n 4'"
+  tree-builder-args: "'-ninit 10 -n 4 -me 0.05'"
+  override_default_args: true
```

Comment:
The mix of `-` and `_` in parameter names right next to each other seems a bit off. Looking at the rest of this file, `_` is used for everything but `tree-builder-args`. It's probably beyond this PR to change that, so probably fine to use underscores for `override_default_args` here to follow broader convention.